### PR TITLE
Set a Spanner client log

### PR DESCRIPTION
**Pull request by shuheiktgw**
## What this PR does / Why we need it
Set a Spanner client log

## Which issue(s) this PR fixes

Fixes https://github.com/mercari/spanner-autoscaler/issues/38

Thanks for your review!

### CODE REVIEW

File: pkg/metrics/metrics.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"time"
 
-	// nolint:gocritic // TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update
+	// TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update
```
Comment (shuheiktgw) [inline pkg/metrics/metrics.go:9]: I've installed the latest golangci-lint (v1.44.0) by running `make lint` and it fails with the following message so I fixed it by removing the unnecessary `nolint`

```
pkg/metrics/metrics.go:9:2: directive `// nolint:gocritic // TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update` is unused for linter "gocritic" (nolintlint)
	// nolint:gocritic // TODO: Lint suggests that this api is deprecated in favor of 'monitoring/v2', confirm and update
	^
make: *** [lint] Error 1
```

Comment (tjun) [reply pkg/metrics/metrics.go:9]: How about adding this on #63 first?

**Review by rustycl0ck — COMMENTED** [review]
Thanks for making a contribution to this project by sending a bugfix. I left a comment regarding the use of `logging` package and will wait for opinion of other maintainers.

File: pkg/syncer/syncer.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ import (
 	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
 
 	spannerv1alpha1 "github.com/mercari/spanner-autoscaler/api/v1alpha1"
+	"github.com/mercari/spanner-autoscaler/pkg/logging"
```
[the remark this answers is not in the document]
Comment (tjun) [reply pkg/syncer/syncer.go:26]: > But I think we can accept the minor changes in this PR for the sake of closing the open issue/bug for now. 

I agree with this.

[the remark this answers is not in the document]
Comment (tkuchiki) [reply pkg/syncer/syncer.go:26]: >We have a plan to remove this package in future. But I think we can accept the minor changes in this PR for the sake of closing the open issue/bug for now

Yes, I think that's a good idea.
Removing the package is a big change, so let's fix it in another PR.

**Review by rustycl0ck — APPROVED** [review]

**Pull request merged**